README.md

```diff
@@ -10,49 +11,50 @@ This is the easiest way to integrate Rainforest with your deploy scripts or CI s
 
 ## Installation
 
-You can install rainforest-cli with the [gem](https://rubygems.org/) utility.
+Follow the directions on our [download page](https://dl.equinox.io/rainforest_qa/rainforest-cli/stable).
 
-```bash
-gem install rainforest-cli
-```
+The CLI will check for updates and automatically update itself on every use unless the
+`--skip-update` global flag is given.
 
-Alternatively, you can add to your Gemfile if you're in a ruby project. This is *not recommended* for most users. The reason being that we update this gem frequently and you usually want to ensure you have the latest version.
-
-```ruby
-gem "rainforest-cli", require: false
-```
+For migration directions from 1.x, please read our [migration guide](./migration.md).
 
 ## Basic Usage
-To use the cli client, you'll need your API token from a test settings page from inside [Rainforest](https://app.rainforestqa.com/).
+To use the cli client, you'll need your API token from your [integrations settings page](https://app.rainforestqa.com/settings/integrations).
 
-You can either pass the token with `--token YOUR_TOKEN_HERE` CLI option, or put it in the `RAINFOREST_API_TOKEN` environment variable.
+In order to access your account from the CLI, set the `RAINFOREST_API_TOKEN` environment variable
+to your API token. Alternatively, you may pass your token with the global `--token` flag.
+
+CLI Commands are formatted as follows:
+```bash
+rainforest [global flags] <command> [arguments] [command-specific-flags]
```

Comment:
it's `rainforest [global flags] <command> [command-specific-flags]  [arguments]` :)